### PR TITLE
[Feature/52908, 52914, 53379, 53082] Arabic and Urdu support

### PR DESCRIPTION
**Pull request by jonasofrich**
- Changed start page to new design
- Added language selection page
- Added Arabic and Urdu support
- Implemented support for RTL layout

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Configuration/Conf.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class Conf
         //The timeout value takes this into consideration.
         public static int DEFAULT_TIMEOUT_SERVICECALLS_SECONDS => 40;
         public static string DEFAULT_LANGUAGE = "nb"; //In case the device is set to use an unsupported language
-        public static string[] SUPPORTED_LANGUAGES = new string[] { "en", "nb", "nn", "pl", "so", "ti" }; // TODO
+        public static string[] SUPPORTED_LANGUAGES = new string[] { "en", "nb", "nn", "pl", "so", "ti", "ar", "ur" }; // TODO
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Configuration/Conf.cs:30]: Remove // TODO

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 2]
```diff
@@ -1,7 +1,5 @@
 {
-  "LAUNCHER_PAGE_START_BTN_NB": "Start på bokmål",
-  "LAUNCHER_PAGE_START_BTN_NN": "Start på nynorsk",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "Continue in English",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:2]: Have you updated the excel sheet AND the wiki page accordingly?

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 3]
```diff
@@ -1,7 +1,5 @@
 ﻿{
-  "LAUNCHER_PAGE_START_BTN_NB": "Start på bokmål",
-  "LAUNCHER_PAGE_START_BTN_NN": "Start på nynorsk",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "Continue in English",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:2]: Same as above

File: NDB.Covid19/NDB.Covid19/Locales/nn.json [hunk 4]
```diff
@@ -1,7 +1,5 @@
 ﻿{
-  "LAUNCHER_PAGE_START_BTN_NB": "Start på bokmål",
-  "LAUNCHER_PAGE_START_BTN_NN": "Start på nynorsk",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "Continue in English",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nn.json:2]: Same as above

File: NDB.Covid19/NDB.Covid19/Locales/pl.json [hunk 5]
```diff
@@ -1,7 +1,5 @@
 {
-  "LAUNCHER_PAGE_START_BTN_NB": "wewoź żr óćósko",
-  "LAUNCHER_PAGE_START_BTN_NN": "ujzwą hr ąijołęń",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "uneąnnsw mw ówęłąfu",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/pl.json:2]: Same as above

File: NDB.Covid19/NDB.Covid19/Locales/so.json [hunk 6]
```diff
@@ -1,7 +1,5 @@
 ﻿{
-  "LAUNCHER_PAGE_START_BTN_NB": "Start på bokmål",
-  "LAUNCHER_PAGE_START_BTN_NN": "Start på nynorsk",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "Continue in English",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/so.json:2]: Same as above

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19/Configuration/Conf.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Configuration/Conf.cs]: done

File: NDB.Covid19/NDB.Covid19/Locales/en.json (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/en.json:2]: Yes

**Review by Frederik-S-N-dk — APPROVED** [review]

File: NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageThreeFragment.cs [hunk 7]
```diff
@@ -27,6 +27,10 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
 
             infoBoxBody.ContentDescription = WelcomeViewModel.WELCOME_PAGE_THREE_INFOBOX_BODY;
 
+            view.LayoutDirection = LayoutUtils.GetLayoutDirection();
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageThreeFragment.cs:30]: you have the same code in 4 classes. maybe move it to WelcomePageTools

**Review by nc-kano — APPROVED** [review]

**Pull request merged**